README.md

```diff
@@ -161,6 +161,23 @@ as shown below:
 Online refund logic should be implemented if you need it.
 As the first try for the possible customization, you can check out `Sylius\RefundPlugin\Event\UnitsRefunded` event.
 
+## Post-refunding process
+
+After units are refunded, there are multiple other processes that should be triggered. By default, after units refund, there should be **CreditMemo** and
+**RefundPayment** generated. As they're strictly coupled with each other, **RefundPayment** is always created after the **CreditMemo**. Moreover, if **RefundPayment**
+fails, related **CreditMemo** should not be created as well.
+
+`\Sylius\RefundPlugin\ProcessManager\UnitsRefundedProcessManager` service facilitates the whole process. If you want to add one or more steps to it, you should create
+a service implementing `\Sylius\RefundPlugin\ProcessManager\UnitsRefundedProcessStepInterface`, and register if with proper tag:
+
+```xml
+    <service id="App\ProcessManager\CustomAfterRefundProcessManager">
+        <tag name="sylius_refund.units_refunded.process_step" priority="0" />
+    </service>
+```
```

Comment:
Is a xml a way to go here or we should add yaml 🤔  ?

Reply:
Could be yaml 👌